codewof/programming/content/en/pass-the-parcel/question.md

```diff
@@ -0,0 +1,7 @@
+# Pass the parcel
+
+The following function `pass_the_parcel(parcel)` takes a list of strings (each item in the parcel, from inside to out), and should **print** `Person 1 got X!`, where `X` is the *last* item.
```

Comment:
@Lucy is iterating backwards through a list an appropriate difficulty level?

I like this question a lot, but I think it might be more suitable if instead the list went from the outside present to the inside one. That way they don't need to iterate backwards, and the question explanation will probably be simpler.

Reply:
I think that the combination of iterating backwards through the loop, combined with also keeping track of the person number makes it too hard. 

I would agree that having a list that goes from outside to in would be more suitable.

Reply:
Will change